app/helpers/active_storage/meta_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveStorage::MetaHelper
+  def active_storage_meta_tags
+    safe_join [
+      tag.meta(name: 'direct-uploads-url', content: rails_direct_uploads_url)
+    ]
```

Comment:
Currently, yes. Leaves us open to adding more without introducing a name change [like `csrf_meta_tags`](https://github.com/rails/rails/blob/9c35bf2a6a27431c6aa283db781c19f61c5155be/actionview/lib/action_view/helpers/csrf_helper.rb#L31-L32).